themes/startup-theme-components/templates/SilverStripe/StartupThemeComponents/Elemental/Block/ImageTextBlock.ss

```diff
@@ -0,0 +1,27 @@
+<div class="image-text-block">
+    <div class="container image-text-block__container $VariantClass">
+        <div class="image-text-block__left-column">
+            <div class="image-text-block__image-wrap">
+                <img src="$ImageTextBlockImage.URL" class="image-text-block__image" alt="$ImageTextBlockImage.AltText"
+                     loading="lazy"/>
+            </div>
+        </div>
+        <div class="image-text-block__right-column">
+            <% if $Title && $ShowTitle %>
+                <% if $Content %>
+                    <h2>$Title</h2>
+                <% else %>
+                    <h1>$Title</h1>
+                <% end_if %>
+            <% end_if %>
+            <% if $Content %>
+                <div class="image-text-block__content">
+                    $Content
+                </div>
+            <% end_if %>
+            <% if $CTAButtonLink %>
+                <button class="button image-text-block__button">$CTAButtonLink</button>
```

Comment:
Button elements can't be used like this, so you will need to code the full HTML for the link. 

Example below:
https://github.com/silverstripeltd/startup-theme-components/blob/1dab17aa6ae926f036906fcdae4e6518bf5e7c91/themes/startup-theme-components/templates/Includes/Header.ss#L30-L35

Reply:
Yep, Than you @BrookeNZ